packages/api/endpoints/pdrs.js

```diff
@@ -63,25 +65,27 @@ async function del(req, res) {
 
   const pdrS3Key = `${process.env.stackName}/pdrs/${pdrName}`;
 
-  await deleteS3Object(process.env.system_bucket, pdrS3Key);
-
   const pdrModel = new models.Pdr();
+  const pdrPgModel = new PdrPgModel();
+  const knex = await getKnexClient();
 
-  try {
+  await knex.transaction(async (trx) => {
+    await pdrPgModel.delete(trx, { name: pdrName });
     await pdrModel.delete({ pdrName });
+    await deleteS3Object(process.env.system_bucket, pdrS3Key);
+  }).catch((error) => {
+    res.boom.badRequest(`Error deleting PDR ${pdrName}`, error);
+    throw error;
+  });
```

Comment:
I'm nitpicking, but I'd prefer to use try/catch for consistency with other code. Also, I'm not sure what the effect of doing this is

```
res.boom.badRequest(`Error deleting PDR ${pdrName}`, error);
throw error
```

You need a `return` in front of `res.boom.badRequest` to avoid getting to `throw error`, I believe. But we're not trapping any specific error before we `throw`, so I'm wondering what the intent of this code is

Reply:
Undid problematic parts of this change.